src/main/java/edu/gatech/cs2340/trydent/Animation.java

```diff
@@ -0,0 +1,39 @@
+package edu.gatech.cs2340.trydent;
```

Comment:
It probably should be; I think my idea was originally that it'd be in the main package to avoid users having to import as many packages, but they'd need KeyframeAnimationBuilder anyway so I guess it doesn't matter much.